ksqldb-engine/src/test/java/io/confluent/ksql/services/SandboxedAdminClientTest.java

```diff
@@ -44,7 +44,7 @@ private SandboxedAdminClientTest() {
       return TestMethods.builder(Admin.class)
           .ignore("close")
           .ignore("close", Duration.class)
-          .ignore("close", long.class, TimeUnit.class)
+          .ignoreUnchecked("close", long.class, TimeUnit.class)
```

Comment:
instead of dropping the `ignore`, I'm just switching it to not check that the referenced method actually exists.